### ISSUE TITLE
[A11Y] Focus states missing or insufficient

### ISSUE THREAD

**Issue by josephcoombes** [Uniswap/v3-info#218]
**Bug Description**
When using keyboard navigation it's very hard to see where the focus is.

<img width="629" alt="Screen Shot 2022-04-08 at 10 02 39 AM" src="https://user-images.githubusercontent.com/6978295/162488645-2b6e12bc-bc4b-4d16-bbd7-6d1520da236d.png">

Not obvious when this element is focused:

<img width="405" alt="image" src="https://user-images.githubusercontent.com/6978295/162489136-e9255e69-75a4-47cf-8c8f-41284b433596.png">

Missing focus state:
<img width="353" alt="image" src="https://user-images.githubusercontent.com/6978295/162489758-cea83290-b171-4455-b272-0b0c1d2119bf.png">

This search input should have a focus state. Apart from the blinking cursor you don't really know that it's selected. The large dropdown menu is a bit of an indicator, but it's behaviour is a little inconsistent and confusing:

<img width="857" alt="image" src="https://user-images.githubusercontent.com/6978295/162515904-f8a40a6e-b63f-4487-bfe3-fae4de1c4233.png">

**Steps to Reproduce**

1. Try tabbing around and notice how it's very hard to keep track of where focus is

**Expected Behavior**
There should be visible and consistent focus states on all interactive elements. This is more of a design consideration, but the focus states should be prominent like this:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/6978295/162515407-724fd50b-311f-429a-b6da-8235ec2c77d3.png">
